migrations/versions/2a69a8406f71_add_user_idp_id_attribute.py

```diff
@@ -0,0 +1,31 @@
+"""add user IDP id attribute
+
+Revision ID: 2a69a8406f71
+Revises: cf0c99c08578
+Create Date: 2021-05-04 20:39:27.197143
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '2a69a8406f71'
+down_revision = 'cf0c99c08578'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('user', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('idp_id', sa.String(length=40), nullable=True))
```

Comment:
`length=40` is not enough, from the spec:

https://openid.net/specs/openid-connect-core-1_0.html#IDToken
>  REQUIRED. Subject Identifier. A locally unique and never reassigned identifier within the Issuer for the End-User, which is intended to be consumed by the Client, e.g., 24400320 or AItOawmwtWwcT0k51BayewNvutrJUqsvl6qs7A4. It MUST NOT exceed 255 ASCII characters in length. The sub value is a case sensitive string.

Reply:
Thanks for paying attention there, corrected in d818df2